tests/collection.tests.ts

```diff
@@ -117,26 +115,25 @@ describe('Collection tests', () => {
       })
     })
 
-    // external_details listed as required, but not in dev docs needs fixing
-    // it('can successfully update a specific collection', () => {
-    //   scope.put('/api/sites/multiscreen/test_site/collection/test_collection', (body) => {
-    //     expect(body).to.eql(update_collection_payload)
-    //     return body
-    //   }).reply(204)
-    //   return duda.collections.update({
-    //     name: collection_name,
-    //     site_name: 'test_site',
-    //     current_collection_name: 'test_collection',
-    //     external_details: {
-    //       enabled: true,
-    //       external_id: '1',
-    //       external_endpoint: '2',
-    //       page_item_url_field: '3',
-    //       collection_data_json_path: '4',
-    //       authorization_header_value: '5'
-    //     }
-    //   })
-    // })
+    it('can successfully update a specific collection', () => {
+      scope.put('/api/sites/multiscreen/test_site/collection/test_collection', (body) => {
+        expect(body).to.eql(update_collection_payload)
+        return body
+      }).reply(204)
+      return duda.collections.update({
+        name: collection_name,
+        site_name: 'test_site',
+        current_collection_name: 'test_collection',
+        external_details: {
```

Comment:
instead of duplicating these props you can now do this:

```
external_details: { ...external_details }
```

This will duplicate the external_details object above into a new object. (Technically we don't even need to duplicate it, but it's a personal preference). Then you don't have to worry about changing the props in two places, if it comes up.